src/wallet/wallet2.h

```diff
@@ -1137,6 +1142,7 @@ namespace tools
     NodeRPCProxy m_node_rpc_proxy;
     std::unordered_set<crypto::hash> m_scanned_pool_txs[2];
     size_t m_subaddress_lookahead_major, m_subaddress_lookahead_minor;
+    bool m_create_address_file;
```

Comment:
Because wallet creation is only a one-time operation, the introduction of this member data doesn't make sense to me; it can simply be a local variable.

Reply:
OK, but add a flag to the json fo generate_from_json, and make sure there is no prompt in that case.

Reply:
I am against adding another interactive step for a user to follow in the wallet creation process. This just makes it more complicated to get started and I reckon a lot of first-time users are not interested in such an address file anyway.

Reply:
I'm OK with not doing it interactively.

Reply:
In that case I'm fine with not doing it at all. New users who aren't expecting it to happen won't be surprised then. Anyone who wants a copy of the address can just use the `address` command in the wallet and paste it into a file if they want to.